locales/en.json

```diff
@@ -1075,6 +1075,12 @@
     "join": "Join Our Community",
     "subscribeLabel": "Subscribe"
   },
+  "collectionCard": {
+    "volume": "Volume",
+    "highestSale": "Highest Sale",
+    "items": "Items"
```

Comment:
use `totalItems`

Reply:
the translation word for `totalItems` is `Σ Items`, different from that on the figma. I'm not sure if use it.

Reply:
meh.. it's the same, I ask you to use the keys already present as some have already been translated no need to duplicate them

Reply:
I have removed `highestSale` key in the latest commit. I guess refreshing the page could see.